app/src/main/java/edu/wpi/first/shuffleboard/app/components/Tile.java

```diff
@@ -49,14 +52,19 @@ protected Tile() {
 
     getStyleClass().addAll("tile", "card");
     PropertyUtils.bindWithConverter(idProperty(), contentProperty(), w -> "tile[" + w + "]");
-    ((EditableLabel) lookup("#titleLabel")).textProperty().bindBidirectional(
-        EasyBind.monadic(contentProperty()).selectProperty(Component::titleProperty)
-    );
+    titleProperty = EasyBind.monadic(contentProperty()).selectProperty(Component::titleProperty);
+    ((EditableLabel) lookup("#titleLabel")).textProperty().bindBidirectional(titleProperty);
     ((Label) lookup("#titleLabel").lookup(".label")).setTextOverrun(OverrunStyle.LEADING_ELLIPSIS);
     contentView.addListener((__, oldContent, newContent) -> {
       getContentPane()
           .map(Pane::getChildren)
-          .ifPresent(c -> c.setAll(newContent));
+          .ifPresent(c -> {
+            if (newContent != null) {
+              c.setAll(newContent);
```

Comment:
fixes NPE when wrapping widgets in layout.

Reply:
Already fixed in #279